test/regressions/main.tsx

```diff
@@ -161,32 +159,19 @@ function App(props) {
     };
   }, []);
 
-  // Using <link rel="stylesheet" /> does not apply the google Roboto font in chromium headless/headfull.
-  const [fontState, setFontState] = React.useState('pending');
-  React.useEffect(() => {
-    webfontloader.load({
-      custom: {
-        families: ['Unica 77'],
-        urls: ['../../docs/src/styles.css'],
-      },
-      timeout: 20000,
-      active: () => {
-        setFontState('active');
-      },
-      inactive: () => {
-        setFontState('inactive');
-      },
-    });
-  }, []);
```

Comment:
We don't need this as we are not loading webfonts, and the inner `TestViewer` already waits for font load events